rest/utilities_testing.go

```diff
@@ -1386,6 +1386,31 @@ func (bt *BlipTester) SendRev(docId, docRev string, body []byte, properties blip
 
 }
 
+// Expects adminChannels of the form `admin_channels":["alpha"]`
+// If scope and collection are non-zero, builds collection access string for the collection
+func AdminChannelGrant(collection *db.DatabaseCollection, adminChannels string) (collectionAdminChannels string) {
```

Comment:
I like this helper function, but I think it would be preferable to have this return a marshalled PrincipalConfig, instead of building the string manually, if the motivation is to build a payload for the _user or _role API.  That will better support future modifications to PrincipalConfig.

Reply:
I'd also suggest taking the input channel parameter as just []string, to simplify usage.

Reply:
I think let's make this a test function that is like

`func GetUserPayload(username, password string, channels []string) string` 

and 

`func GetRolePayload(rolename string, channels []string) string` 

if they share common functionality, which I'm not sure they do, we can do a private function like

`func addChannelsToPrincipal(config *PrincipalConfig, channels []string)`

that does some common work but then returns the marshalled output of the entire payload.